### DISCUSSION TITLE
when I share a project with Github via phpstorm I get an error: “access to this site is restricted”

### DISCUSSION THREAD

**Discussion by KirillDolbnya** [community/community#145294] [Programming Help]
### Body

when I share a project with Github via phpstorm I get an error: “access to this site is restricted”

### Guidelines

- [ ] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by denis23x** [comment]
Hi @KirillDolbnya 

Probably you faced with authentication problems. Try to remove your GitHub account and add it again via personal access token.

Here you can find instructions how to do that https://www.jetbrains.com/help/phpstorm/github.html

**Comment by mecodeatlas** [comment]
Hi @KirillDolbnya 👋🏾,It looks like @denis23x's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
